src/controllers/controlpickermenu.cpp

```diff
@@ -28,6 +28,19 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     m_buttonParameterStr = tr("Button Parameter %1");
     m_libraryStr = tr("Library");
 
+    m_numGroupsTrMap.insert("Channel", m_deckStr);
+    m_numGroupsTrMap.insert("Sampler", m_samplerStr);
+    m_numGroupsTrMap.insert("PreviewDeck", m_previewdeckStr);
+    m_numGroupsTrMap.insert("Microphone", m_microphoneStr);
+    m_numGroupsTrMap.insert("Auxiliary", m_auxStr);
+    m_numGroupsTrMap.insert("EffectRack", m_effectRackStr);
+
+    m_otherGroupsTrMap.insert("Skin", tr("Skin"));
+    m_otherGroupsTrMap.insert("Library", m_libraryStr);
+    m_otherGroupsTrMap.insert("Controller", tr("Controller"));
+    // TODO(ronso0) "translate" legacy 'Master' to 'Main' in main branch?
+    m_otherGroupsTrMap.insert("Master", "Master");
```

Comment:
This PR is against the main branch. Or did you plan to merge this to 2.3?

Reply:
oh, initially I did target 2.3, then it reached a level where I introduced new (though redundant) translations translations (which I didn't want to do in the stable branch), so I rebased onto main, then I found a solution to reuse existing translations... and forgot to rebase onto 2.3
will do so now.

Reply:
done.